packages/_releases.njk

```diff
@@ -1,3 +1,9 @@
+{% import "_includes/install_chart.njk" as charts %}
+
+<div class="chart-container">
+  {{ charts.chart(pkg.installs_weekly, pkg.removals_weekly, pkg.weekly_dates, pkg.upgrades_weekly) }}
```

Comment:
Just a thought, not asking you to rewrite everything, but you could also pass that data to a filter, so that you can implement the rendering code in JS. That would get you niceties like functions, scope, static analysis ...